ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -788,8 +765,6 @@ public void onCompletion(ReadableStreamChannel routerResult, Exception routerExc
     public void onCompletion(String routerResult, Exception routerException) {
       callbackTracker.markOperationEnd();
       if (routerResult == null && routerException == null) {
-        frontendMetrics.routerCallbackError.inc();
-        callbackTracker.markCallbackProcessingEnd();
```

Comment:
Are there any adverse affects to not marking the end time in cases like this? Will this exception just stop execution completely, since it signifies a programming error.